Dockerfile

```diff
@@ -13,7 +13,7 @@ FROM alpine:3.19 as frontend-builder-1
 COPY client/dist /frontend/client/dist
 
 # MODE 2: build static content in docker, can be used for a local development
-FROM node:18.1-bookworm as frontend-builder-2
+FROM node:20-bookworm as frontend-builder-2
```

Comment:
Is this strictly needed?

Reply:
package.json allows using node 20, so there should be no issues with this

Reply:
Does everything work correctly with Node 20 though?

I'm pretty sure @arikfr mentioned (ages ago) there were issues with some recent version of Node.

Personally, I have no idea if that's still a problem or if it's been solved already.  That's why I'm asking. :smile: